### PR TITLE
so2 volcano story

### PR DESCRIPTION
**Pull request by hanbyul-here**
Add so2 volcano story (air quality)

### CODE REVIEW

**Comment by aboydnw** [comment]
@kcarini I just requested reviews from Daniel and Erik. I'd say if this is good to go then we JFDI it. I can ask for forgiveness later if something ends up being super wrong

**Comment by hanbyul-here** [comment]
I noticed storytelling map jumps instead animates between the points. Did we change the behavior of the component at one point? or is this the bug that we didn't catch before? @danielfdsilva ?  
(This behavior was not introduced by this pr, and should not be a blocker)

**Comment by kcarini** [comment]
> I noticed storytelling map jumps instead animates between the points. Did we change the behavior of the component at one point? or is this the bug that we didn't catch before? @danielfdsilva ? (This behavior was not introduced by this pr, and should not be a blocker)

I tried adjusting some of the field attributes in this scrollytelling block and wasn't noticing the changes being reflected in my local instance.

**Comment by kcarini** [comment]
> @kcarini I just requested reviews from Daniel and Erik. I'd say if this is good to go then we JFDI it. I can ask for forgiveness later if something ends up being super wrong

I fixed a typo that I saw in a caption, moved the comparison map to a different spot, and tried some adjusting of the scrollytelling attributes but those didn't seem to work. As-is the scrollytelling block is purely aesthetic in my opinion, the locations aren't mentioned in the captions.

**Review by nerik — APPROVED** [review]
Looks good to merge functionality-wise.

I'm not able to reproduce the map issue mentioned by @hanbyul-here. 

Re content: apart from the lack of correlation between text and map as mentioned by @kcarini, I also don't see particularly visible outliers or remarkable values in the locations where the map centers. Just wondering, how were these locations selected?

**Review by danielfdsilva — CHANGES_REQUESTED** [review]
There are some images that are too large. I can look into this in a bit

**Comment by nerik** [comment]
Please let me know once that gets merged so I can scrape the accidentally committed MB token from the git history.

**Review by danielfdsilva — APPROVED** [review]
Images updated

**Comment by aboydnw** [comment]
@danielfdsilva @hanbyul-here can you check @kcarini 's scrollytelling configuration? Sounds like she was trying to adjust to location show in each frame but it didn't change. Looking at the preview, it hovers over the same spots as before she made her changes. Once we sort that out, I think this is ready to merge.

**Comment by kcarini** [comment]
> 

I looked at the scrollytelling block again and I have a better understanding of the formatting. I actually used the SO2 Dataset Exploration to help me quickly identify some years and locations to highlight in the scrollytelling block. While the locations aren't directly related to the text, I added a sentence in each chapter text describing the location and year. I don't think that people will necessarily notice the year changing as they scroll. It may be a best practice to not change the year of a dataset in the scrollytelling block unless we can make is more obvious. But for now I reiterated it in the text. 

For some reason the dataset is not rendering in the chapter for 2009 centered on the Kamchatka Peninsula.

**Comment by aboydnw** [comment]
Thanks @kcarini ! I'm not sure why that 2009 frame isn't showing up either. I'll let @danielfdsilva @hanbyul-here troubleshoot.

I did just make the discovery a featured discovery. I think most the discoveries we publish at this point should be featured discoveries. Until we reach a point where we have a) lots of content or b) another way for us to showcase "recently added" discoveries like IMPACT has asked for. This is a little lower priority than some of the other work we have on deck, though, so I propose this as a standard practice until then.

File: discoveries/air-quality-so2-volcanoes.discoveries.mdx
Comment (danielfdsilva) [inline discoveries/air-quality-so2-volcanoes.discoveries.mdx:64]: @kcarini @aboydnw The 2009 dataset is not loading because the date is incorrect. Being an yearly dataset, the data falls on the 1st of january, so the date should be `2009-01-01`, like you've one with all the other chapters.

```suggestion
    datetime='2009-01-01'
```

**Review by danielfdsilva — CHANGES_REQUESTED** [review]

**Comment by danielfdsilva** [comment]
@kcarini I think you're right that the changes are not super clear, especially when moving to a different place and loading different data. The text should help describe what's on the map. Ideally the map could be removed and the text would still make sense.

**Comment by aboydnw** [comment]
Once these changes are made, we got approval from Dee-Dee to push to production, so this can be including in our next production push @danielfdsilva

**Comment by kcarini** [comment]
@danielfdsilva @aboydnw I've made DeeDee's requested changes, so I think it is good to go.

File: discoveries/air-quality-so2-volcanoes.discoveries.mdx
Comment (hanbyul-here) [inline discoveries/air-quality-so2-volcanoes.discoveries.mdx:109]: Is this highlight necessary? It looks like the entire timeline is selected. @kcarini

Comment (kcarini) [reply discoveries/air-quality-so2-volcanoes.discoveries.mdx:109]: no idea. i just copied formatting for charts from the doc. is `highlightStart` only required if your chart is for a subset of the dataset? should line 109 be removed entirely, is that what you mean? @hanbyul-here 

Screen Shot 2023-01-23 at 10 07 39 AM

Comment (hanbyul-here) [reply discoveries/air-quality-so2-volcanoes.discoveries.mdx:109]: `highlight` related properties are only required if the chart has a reference/highlight area.  (area with sky blue background). It sounds like this chart doesn't need one, we can get rid of the lien from 109 to 111, 
Screen Shot 2023-01-23 at 12 44 23 PM

File: discoveries/no2-and-so2.discoveries.mdx
Comment (hanbyul-here) [inline discoveries/no2-and-so2.discoveries.mdx:114]: Just want to make sure if this change was intended?

Comment (kcarini) [reply discoveries/no2-and-so2.discoveries.mdx:114]: oops. i must have edited the wrong .mdx. I changed it back.

File: discoveries/air-quality-so2-volcanoes.discoveries.mdx
Comment (danielfdsilva) [inline discoveries/air-quality-so2-volcanoes.discoveries.mdx:13]: Maybe update to today?

```suggestion
pubDate: 2023-01-25
```

**Review by danielfdsilva — APPROVED** [review]

**Pull request merged**
